common/src/main/java/org/apache/rocketmq/common/Configuration.java

```diff
@@ -219,6 +224,35 @@ public void persist() {
         }
     }
 
+    public void persistBrokerConf(Properties from) {
+        BufferedReader reader = null;
+        try {
+            readWriteLock.readLock().lockInterruptibly();
```

Comment:
The following inline method already exists in getAllConfigsFormatString Is it possible to take the relevant logic?